[2024-09-10T08:13:18.802+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T08:13:18.906+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V5.third_task scheduled__2022-03-06T00:00:00+00:00 [queued]>
[2024-09-10T08:13:18.976+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V5.third_task scheduled__2022-03-06T00:00:00+00:00 [queued]>
[2024-09-10T08:13:18.986+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-10T08:13:19.110+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): third_task> on 2022-03-06 00:00:00+00:00
[2024-09-10T08:13:19.142+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V5', 'third_task', 'scheduled__2022-03-06T00:00:00+00:00', '--job-id', '7061', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpmv72eiog']
[2024-09-10T08:13:19.157+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=31103) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-10T08:13:19.164+0000] {standard_task_runner.py:72} INFO - Started process 31127 to run task
[2024-09-10T08:13:19.156+0000] {standard_task_runner.py:105} INFO - Job 7061: Subtask third_task
[2024-09-10T08:13:19.505+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_V5.third_task scheduled__2022-03-06T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-10T08:13:20.060+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_V5' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2022-03-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-06T00:00:00+00:00'
[2024-09-10T08:13:20.070+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T08:13:20.153+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-10T08:13:20.205+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is the third task!']
[2024-09-10T08:13:20.255+0000] {subprocess.py:86} INFO - Output:
[2024-09-10T08:13:20.315+0000] {subprocess.py:93} INFO - hey, this is the third task!
[2024-09-10T08:13:20.333+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-10T08:13:20.509+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T08:13:20.525+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V5, task_id=third_task, run_id=scheduled__2022-03-06T00:00:00+00:00, execution_date=20220306T000000, start_date=20240910T081318, end_date=20240910T081320
[2024-09-10T08:13:20.805+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T08:13:20.867+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
